admin/release.py

```diff
@@ -510,23 +514,28 @@ def upload_rpms(scratch_directory, target_bucket, version, build_server):
         target_distro_suffix = ""
 
     operating_systems = [
-        {'distro': 'fedora', 'version': '20', 'arch': 'x86_64'},
-        {'distro': 'centos', 'version': '7', 'arch': 'x86_64'},
-        {'distro': 'ubuntu', 'version': '14.04', 'arch': 'amd64'},
+        {'distro': 'fedora', 'version': '20'},
+        {'distro': 'centos', 'version': '7'},
+        {'distro': 'ubuntu', 'version': '14.04'},
+        {'distro': 'ubuntu', 'version': '15.04'},
     ]
 
     for operating_system in operating_systems:
+        for package_type, distribution_names in PACKAGE_TYPE_MAP.items():
+            if operating_system['distro'] in distribution_names:
+                architecture = ARCH['native'][package_type]
```

Comment:
This change will make it more difficult to support additional architectures later.  I'd suggest adding `arch` back into `operating_systems`.

Instead of naming package-dependent values, add `'arch': 'native'` for each OS. Later, we can do a global replace of `native` into `amd64`. Then we can add `i386` alternatives pretty simply.

This line is then `architecture = ARCH[operating_system['arch']][package_type]`